src/components/EventList.vue

```diff
@@ -4,6 +4,15 @@
                 :key="event.id"
                 :event="event"
                 ></event-list-item>
+    <div v-if="!events.length">
+      <p>
+        No results in your search range.
+      </p>
+      <p @click="expandSearch">
```

Comment:
can you make this a button instead of a paragraph?

Reply:
https://github.com/SFDigitalServices/neighborhood-noticing/pull/17/commits/10a945358a07e63bf448d7b45fa048c654716829